### PR TITLE
Update all device tracker configuration variables

### PR DESCRIPTION
**Pull request by Mryck**
**Description:**

Update of 25 device tracker configuration variables to follow the new standards  #6385

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/device_tracker.ubus.markdown [hunk 1]
```diff
@@ -63,12 +63,25 @@ device_tracker:
...
+  required: true
+  type: string
+password:
+  description: The password for your given admin account.
+  required: true
+  type: string
+dhcp_software:
+  description: The DHCP software used in your router: `dnsmasq`, `dhcpd`, or `none`.
+  required: false
+  default: `dnsmasq`
```
Comment (klaasnicolaas) [inline source/_components/device_tracker.ubus.markdown]: Leave the `` away or put it between "" like: "`dnsmasq`"

File: source/_components/device_tracker.upc_connect.markdown [hunk 2]
```diff
@@ -23,14 +23,17 @@ device_tracker:
...
 ```
 
-Configuration variables:
-
-- **host** (*Optional*): The IP address of your router. Set it if you are not using `192.168.0.1`.
+{% configuration %}
+host:
+  description: The IP address of your router.
+  required: false
+  default: `192.168.0.1`
```
Comment (klaasnicolaas) [inline source/_components/device_tracker.upc_connect.markdown]: Same

File: source/_components/device_tracker.swisscom.markdown [hunk 3]
```diff
@@ -27,8 +27,12 @@ device_tracker:
...
 ```
 
-Configuration variables:
-
-- **host** (*Optional*): The IP address of your router. Set it if you are not using `192.168.1.1`.
+{% configuration %}
+host:
+  description: The IP address of your router.
+  required: false
+  default: `192.168.1.1`
```
Comment (klaasnicolaas) [inline source/_components/device_tracker.swisscom.markdown]: Same

File: source/_components/device_tracker.sky_hub.markdown [hunk 4]
```diff
@@ -23,9 +23,12 @@ device_tracker:
...
 ```
 
-Configuration variables:
-
-- **host** (*Optional*): The IP address of your router. Defaults to `192.168.1.254`.
+{% configuration %}
+host:
+  description: The IP address of your router.
+  required: false
+  default: `192.168.1.254`
```
Comment (klaasnicolaas) [inline source/_components/device_tracker.sky_hub.markdown]: Same

File: source/_components/device_tracker.owntracks.markdown [hunk 5]
```diff
@@ -25,15 +25,39 @@ device_tracker:
...
+  default: All users who are connected to Home Assistant via Owntracks.
+  type: list
+secret:
+  description: [Payload encryption key](http://owntracks.org/booklet/features/encrypt/). This is usable when communicating with a third-party untrusted server or a public server (where anybody can subscribe to any topic). By default the payload is assumed to be unencrypted (although the communication between Home Assistant and the server might still be encrypted). This feature requires the `libsodium` library to be present.
+  required: false
+  type: string
+mqtt_topic:
+  description: The topic to subscribe for Owntracks updates on your MQTT instance.
+  required: false
+  default: `owntracks/#`
```
Comment (klaasnicolaas) [inline source/_components/device_tracker.owntracks.markdown]: Same

**Comment by Mryck** [comment]
Thanks @klaasnicolaas it should be fine now ! 👍

File: source/_components/device_tracker.automatic.markdown [hunk 6]
```diff
@@ -37,20 +37,31 @@ device_tracker:
...
+  required: true
+  type: string
+secret:
+  description: "The OAuth client secret (get from https://developer.automatic.com/)."
+  required: true
+  type: string
+current_location:
+  description: "Set to `true` if you have requested `scope:current_location` for your account. Home Assistant will then be able to receive periodic location updates during trips."
+  required: false
+  type: boolean
```
Comment (frenck) [inline source/_components/device_tracker.automatic.markdown:53]: Type `boolean` is missing `default`.

File: source/_components/device_tracker.netgear.markdown [hunk 7]
```diff
@@ -26,16 +26,41 @@ device_tracker:
...
+{% configuration %}
+url:
+  description: The base URL, e.g., `http://routerlogin.com:5000` for example. If not provided `host` and `port` are used. If none provided autodetection of the URL will be used.
+  required: false
+  type: string
+host:
+  description: The IP address of your router, e.g., `192.168.1.1`.
+  required: false
+  type: string
+port:
```
Comment (frenck) [inline source/_components/device_tracker.netgear.markdown:38]: Not really clear by the older docs and code, but the default here is actually `5000`.
Please add the `default` key.

**Review by frenck — CHANGES_REQUESTED** [review]
Nice @Mryck!
I've found 2 little things, could you please take a look? 👍 

You really did a fine job btw. I love the way you even updated small details (Like `eg.` -> `e.g.,` and many others).

One other note though, this PR is kinda hitting the size limit 😉 
A couple of smaller PR's is usually easier to review/merge/process and reduces the risk of merge conflicts.

**Comment by Mryck** [comment]
Thanks @frenck for the review,  I agree about the size of the PR I will be carful, I made the change that you've requested 👍

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for updating @Mryck! 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**